.travis.yml

```diff
@@ -18,4 +17,6 @@ script:
 
 matrix:
   allow_failures:
-    - php: hhvm
+    - php:
+      - hhvm
```

Comment:
hhvm is not compatible to PHP any more.
We should remove hhvm.
https://hhvm.com/blog/2019/02/11/hhvm-4.0.0.html